src/datepicker/calendar-header.js

```diff
@@ -97,6 +102,26 @@ export default class CalendarHeader extends React.Component<
     }
   };
 
+  isHiddenPaginationButton = (direction: $Values<typeof DIRECTION>) => {
```

Comment:
is this better described as 'disabled` rather than hidden?

Reply:
I think this might be confusing because on line 138 I also set this value to determine if the button is disabled, but this actually does control if the arrow is hidden

Reply:
Can you share reproduction steps for when the arrow is hidden? I clicked the right arrow until 'December 2030' is on the right-hand side. This displays a grey arrow and disabled button. Based on this [line](https://github.com/uber-web/baseui/pull/1630/files#diff-92ab803c37c9efa8648d2fa9682bc9c8R227), I would expect the arrow to not be rendered.

What I'm trying to get at is - why was this additional check needed in addition to disabling the button?

Reply:
Following up with a discussion I had with @chasestarr, we decided that the current implementation is the correct solution for the use case here where each month has it's own dedicated header